### PR TITLE
Update server_replay.md

### PR DESCRIPTION
**Pull request by Sup3rFire**

### CODE REVIEW

**Comment by Sup3rFire** [comment]
Sorry if my English was a bit bad

**Comment by Poyo-SSB** [comment]
Thanks for contributing! This needs a little work to meet the standards of this repository. Please make sure to indent things the same way they're indented in other documents (four spaces) and match the conventional style.

Also, please use `integer` or `float` for numeric types, and describe array types by the type of object the array contains (e.g. `number[]`).

File: Messages/server_replay.md [hunk 1]
```diff
@@ -7,5 +7,14 @@ A message containing replay input data for the players currently playing.
...
+         * (integer) `frame`: The frame the action was done on
+         * (string) `type`: Possible values are `"keydown"` or `"keyup"` 
+         * (object) `data`: 
+            * (string) `key`: Possible values are `"hardDrop"`, `"softDrop"`, `"moveLeft"`, `"moveRight"`, `"rotateCW"`, `"rotateCCW"`, `"rotate180"` and `"hold"`  
+            * (number) `subframe`: The subframe the action was done on
+         * (integer) `provisioned`: Value is unknown. Seems like a value that goes up by a random amount every time
 
-TODO: This document is incomplete.
\ No newline at end of file
+TODO: This document is incomplete.
```
Comment (Poyo-SSB) [inline Messages/server_replay.md:20]: If there aren't any undocumented fields, then this can be safely removed.

File: Messages/server_replay.md [hunk 2]
```diff
@@ -7,5 +7,14 @@ A message containing replay input data for the players currently playing.
 * (object):
     * (integer) `id`: [The id of this message.](../Ribbon.md#id-messages)
     * (string) `command`: Always `"replay"`.
+    * (object) `data`: 
+      * (string) `listenID`: The id of the user which the replay is from + gameID
```
Comment (Poyo-SSB) [inline Messages/server_replay.md:11]: This is unclear—could you give an example of what this looks like?

File: Messages/server_replay.md [hunk 3]
```diff
@@ -7,5 +7,12 @@ A message containing replay input data for the players currently playing.
 * (object):
     * (integer) `id`: [The id of this message.](../Ribbon.md#id-messages)
     * (string) `command`: Always `"replay"`.
-
-TODO: This document is incomplete.
\ No newline at end of file
+    * (object) `data`: 
+        * (string) `listenID`: The event stream this data belongs to. This is the id of the user which sent the data concatenated with the id of the current game.
```
Comment (Poyo-SSB) [inline Messages/server_replay.md:11]: Where can the user find the game id? Is it randomly generated by the client or does the server send it at some point?

Comment (Proximitynow19) [reply Messages/server_replay.md:11]: i believe it's found on readymulti or startmulti one of the two

Comment (Sup3rFire) [reply Messages/server_replay.md:11]: It's found on readymulti

File: Messages/server_replay.md [hunk 4]
```diff
@@ -7,5 +7,12 @@ A message containing replay input data for the players currently playing.
...
+        * (object[]) `frames`:
+            * (integer) `frame`: The frame the event occurred on.
+            * (string) `type`: Possible values are `"keydown"` and `"keyup"`.
```
Comment (ZudoB) [inline Messages/server_replay.md:14]: I've observed this to also have `"ige"` as a possible value, in which case the `data`  object below has information about lines sent, garbage well column and the _username_ of the sender

Comment (Sup3rFire) [reply Messages/server_replay.md:14]: Oh yeah, I missed that. I'm gonna just mark the document as incomplete as I don't know much about the values.

**Pull request merged**